packages/ibm-products/src/components/Datagrid/Datagrid/DatagridHeaderRow.tsx

```diff
@@ -252,7 +257,9 @@ const HeaderRow = (
               {...getAccessibilityProps(header)}
             >
               {header.render('Header')}
-              {renderSlug(header.slug)}
+              {header.aiLabel
+                ? renderAILabel(header.aiLabel)
+                : renderAILabel(header.slug)}
```

Comment:
```suggestion
              {renderAILabel(header.aiLabel || header.slug)}
```

Reply:
Thanks @devadula-nandan.
This is updated.